.github/workflows/playwright-docs-check.yml

```diff
@@ -0,0 +1,68 @@
+name: Verify Playwright Documentation
+
+on:
+  pull_request:
+    paths:
+      - 'openmetadata-ui/src/main/resources/ui/playwright/e2e/**/*.spec.ts'
+      - 'openmetadata-ui/src/main/resources/ui/playwright/doc-generator/**'
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  verify-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'openmetadata-ui/src/main/resources/ui/.nvmrc'
+          cache: 'yarn'
+          cache-dependency-path: 'openmetadata-ui/src/main/resources/ui/yarn.lock'
+
+      - name: Install Dependencies
+        working-directory: openmetadata-ui/src/main/resources/ui
+        run: yarn install --frozen-lockfile --ignore-scripts
+
+      - name: Install Playwright Browsers
+        # We need browsers installed for 'playwright test --list' to work in some versions,
+        # although strictly speaking just listing shouldn't require binaries if configured right.
+        # Adding just in case.
+        working-directory: openmetadata-ui/src/main/resources/ui
+        run: npx playwright install --with-deps
+
+      - name: Generate Documentation
+        working-directory: openmetadata-ui/src/main/resources/ui
+        run: node playwright/doc-generator/generate.js
+
+      - name: Check for Changes
+        id: git-check
+        run: |
+          git diff --exit-code --quiet openmetadata-ui/src/main/resources/ui/playwright/docs || echo "changes=true" >> $GITHUB_OUTPUT
+
+      - name: Commit and Push Changes
+        if: steps.git-check.outputs.changes == 'true'
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git add openmetadata-ui/src/main/resources/ui/playwright/docs
+          git commit -m "docs: auto-generate playwright documentation"
+          git push
+
+      - name: Comment on PR
```

Comment:
Which PR this will add comment? since it's running on main?

Reply:
It will run agains each PR, If PR has playwright changes then it will update docs and push into PR.